superset-frontend/src/SqlLab/components/TabbedSqlEditors.jsx

```diff
@@ -18,18 +18,19 @@
  */
 import React from 'react';
 import PropTypes from 'prop-types';
-import { MenuItem, DropdownButton, Tab, Tabs } from 'react-bootstrap';
+import { EditableTabs } from 'src/common/components/Tabs';
+import { Dropdown } from 'src/common/components/Dropdown';
```

Comment:
Nice! A new component has entered the game! Might be nice to see a screenshot of this on the PR.

Reply:
And actually, we should probably make a ticket to restyle this to match our design system, and use it everywhere, replacing the React Bootstrap implementation.